src/config.c

```diff
@@ -2694,6 +2694,7 @@ standardConfig configs[] = {
     /* Other configs */
     createTimeTConfig("repl-backlog-ttl", NULL, MODIFIABLE_CONFIG, 0, LONG_MAX, server.repl_backlog_time_limit, 60*60, INTEGER_CONFIG, NULL, NULL), /* Default: 1 hour */
     createOffTConfig("auto-aof-rewrite-min-size", NULL, MODIFIABLE_CONFIG, 0, LLONG_MAX, server.aof_rewrite_min_size, 64*1024*1024, MEMORY_CONFIG, NULL, NULL),
+    createOffTConfig("loading-process-events-interval-bytes", NULL, MODIFIABLE_CONFIG, 1024, INT_MAX, server.loading_process_events_interval_bytes, 1024*1024*2, INTEGER_CONFIG, NULL, NULL),
```

Comment:
this is not the first time we add a config for testing and don't mention it in redis.conf so not a precedent.
and i don't see any way it can be abused, so this is fine.
there are quite a few other tests in which i had to disable `rdbcompression` and add a lot of data so that redis is responsive during long loading.

maybe in order prevent confusions as to what these are and why they're not documented, we wanna add a new config flag HIDDEN_CONFIG that'll hide them from `CONFIG GET *` (idea for a followup PR maybe)

Reply:
@yossigo WDYT? do we need a major-decision for this new config? i think not.

also state what you think of HIDDEN_CONFIG.